src/Source/Language/PHP/PHPTokenizerInternal.php

```diff
@@ -1008,23 +1008,23 @@ private function tokenize(): void
                     $image = $token[1];
 
                     // Check for a context sensitive alternative
-                    if (isset(self::$alternativeMap[$type][$previousType])) {
-                        $type = self::$alternativeMap[$type][$previousType];
+                    if (isset(self::$alternativeMap[$type][$previousType ?? 0])) {
```

Comment:
Isn't this an idea? (Also on the lines below)
```suggestion
                    if (isset(previousType, self::$alternativeMap[$type][$previousType])) {
```

Reply:
Thanks, per usual when I have a series I'll punt this to the net PR since this one got approved. Just to keep things moving but also not ignoring your suggestion.